RealmModelGenerator/EntitieViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  EntitieViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/21/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+
+class EntitiesViewController: NSViewController, EntitiesViewDelegate, EntitiesViewDataSource {
+    static let TAG = NSStringFromClass(EntitiesViewController)
+    
+    @IBOutlet weak var entitiesView: EntitiesView!
+    
+    private var schema:Schema = Schema()
```

Comment:
this VC shouldn't need a reference to the Schema and entities,
it should only need a reference to the model.